cli/info.rs

```diff
@@ -223,6 +228,95 @@ impl FileInfoDepTree {
   }
 }
 
+/// Flat graph vertex with all shallow dependencies
+#[derive(Serialize)]
+#[serde(rename_all = "camelCase")]
+struct FileInfoVertex {
+  size: usize,
+  total_size: Option<usize>,
```

Comment:
We don't show `total` size of vertex in terminal output and I think we should skip it in JSON as well. It's useful to have total size of _all_ dependencies in top level structure, but calculating sizes of subdependencies should be left to implementors. @lucacasonato WDYT?

Reply:
Yup I agree

Reply:
@bartOssh let's remove `total_size` then. Otherwise LGTM and we can land this PR

Reply:
@bartlomieju Curious about this. I requested the original feature and noticed that the total size of each dependency was left out in the end. I thought this a useful feature to see how big your dependencies were. Of course, you can compute this manually by running `deno info` against each dependency but that's a bit of a pain. I agree that these should be consistent, just curious about the reasoning behind leaving out the total size of dependencies.

Reply:
I have implemented the change for `total_size` that @bartlomieju and @lucacasonato agreed on from vertex of a flat graph, and added `total_size`to module info on the top level so only total size of module will be shown. 
But is you think @cknight suggestion shuld be applied I will cherry pic this and push changes back again.
Thanks

Reply:
> I agree that these should be consistent, just curious about the reasoning behind leaving out the total size of dependencies. 

Circular dependencies can be really confusing, especially if they show up multiple times. Showing only the size of singular dependency is what browsers do. If there will be huge demand for "total" size we can revisit the topic, but for now I think we should go only with size of the concrete module without it's deps.

Reply:
Thanks @bartlomieju, makes sense